script/Common/00_00.txt

```diff
@@ -1172,12 +1178,12 @@ Tracing along one of the lines,
 [sha:59ebd488a7e20616490cd94c00a7d8cf5abc2b43]{
 -- Page 85, Offset 331.
 -- 先生は驚いている。
-Sensei was shocked.
+Sensei is shocked.
 }
 [sha:fe2fc30da2d68e98dfc2d9d8ff85226c77c00788]{
 -- Page 85, Offset 332.
 -- 僕は得意になって、今度は木の根元、横一直線にある線にナイフを差しこんで、
-I got really into it, so I thrust my knife into a horizontal line running along the base of the tree―――
+I get really into it, so I thrust my knife into a horizontal line running along the base of the tree―――
```

Comment:
The link between 'getting really into it' and 'cutting the line tree' isn't very clear imo, as in it doesn't sound like a natural progression of thoughts here. I also think that the line might be a little incorrect. 

My interpretation of the line is 'I'm pretty good at this so I'll try and show something more impressive next, aka tree line cutting'. I associate 'getting into something' with active interest/participation rather than ability, which I feel doesn't quite convey 得意 well.

Reply:
Yeah, it's something like "Feeling proud at display, I trust my knife..."
I think I just assumed 'I get really into it' was supposed to convey that somehow but thinking about it more clearly this line is a bit weird.

Reply:
Thought about this for a while... Maybe splitting it into two could make it more serviceable. A phrase that I think makes more sense but I'm not sure how to fit in is 'getting to my head'.

> Getting carried away, I thrust my knife into a horizontal line running along the base of the tree―――

> Self-satisfied, I look to my next target. I thrust my knife into a horizontal line running along the base of the tree―――